Classes/RZPoseurWebView.m

```diff
@@ -71,6 +71,12 @@ - (NSURLRequest *)request
     return nil;
 }
 
+- (UIScrollView *)scrollView
+{
+    RZPoseurWebViewMethodUnimplementedAssert();
+    return nil;
```

Comment:
If the subclass implementations match, then why not return `self.backingView.scrollView` from `-scrollView` on `RZPoseurWebView`?